src/Schema/Builder/SchemaBuilder.php

```diff
@@ -150,4 +177,13 @@ public function hasEnum($enum): bool
     {
         return $this->hasType($enum);
     }
+
+    private function generateInvalidGraphQLNameWarning(string $type, string $name): void
+    {
+        $message = "Skipping schema generation for %s with identifier '%s'. "
+            . 'Please rename given %s according to GraphQL specification '
+            . '(http://spec.graphql.org/June2018/#sec-Names)';
```

Comment:
Not sure about maintaining the URL within this message - we already have a newer specification release: https://spec.graphql.org/October2021/#sec-Names. It doesn't seem to be tied to any GraphQL package used by DXP, so maybe we can tweak it a little, e.g.:
```suggestion
            . '(http://spec.graphql.org/[latest-release]/#sec-Names)';
```
I see it was added before your change and before the newest release of the specification (https://github.com/ezsystems/ezplatform-graphql/pull/109) but I will personally make it a little tidier anyway. @adamwojs would you agree?

Reply:
Tbh this message as you have noticed was just really copied and ideally, it should never be thrown because the check is done *after* the item renaming. We cannot expect the user to rename this item as this is generated later on when the whole schema is built.

Honestly, I would leave only `Skipping schema generation for %s with identifier '%s'.` as user has no power at this point to update anything.

Reply:
Fine by me. Nonetheless, I would give some more explanation though on why skipping took place. Maybe:
```
Skipping schema generation for %s with identifier '%s' as it stands against GraphQL specification. For more details see http://spec.graphql.org/[latest-release]/#sec-Names.
```

Reply:
Done in https://github.com/ezsystems/ezplatform-graphql/pull/116/commits/01261bac86888d8b447b9d9e41813213ac1b29a6